e2e-tests/portal/cypress/e2e/journeys/maker/submit-issued-facilities-for-review/submit-issued-bond-facility-with-created-field.spec.js

```diff
@@ -36,17 +50,21 @@ context('Issue Bond Form - Submit issued bond with inserted element on page', ()
   });
 
   it("should not insert created element's data into the bond", () => {
+    cy.loginGoToDealPage(BANK1_MAKER1);
     pages.dashboardDeals.visit();
     cy.clickDashboardDealLink();
 
     pages.contract.proceedToReview().should('not.exist');
 
     const bondId = dealFacilities.bonds[0]._id;
-    const bondRow = pages.contract.bondTransactionsTable.row(bondId);
 
-    bondRow.issueFacilityLink().click();
+    pages.contract.bondTransactionsTable.row(bondId).issueFacilityLink().click();
+
+    pages.bondDetails.facilityStageIssuedInput().click();
 
-    cy.completeDateFormFields({ idPrefix: 'issuedDate' });
+    pages.bondDetails.requestedCoverStartDateDayInput().clear().type(startDate.day);
+    pages.bondDetails.requestedCoverStartDateMonthInput().clear().type(startDate.month);
+    pages.bondDetails.requestedCoverStartDateYearInput().clear().type(startDate.year);
```

Comment:
We can use `cy.completeDateFormFields` like below instead of 3x lines

Also don't forget we have a `cy.keyboardInput` that's faster and cleaner than `.clear().type()`

Reply:
Used cy.keyboardInput